src/components/UserInfo.js

```diff
@@ -1,20 +1,27 @@
 export default class UserInfo {
-  constructor({ userNameSelector, userDescriptionSelector }) {
+  constructor({ userNameSelector, userDescriptionSelector, avatarSelector }) {
     this._userNameElement = document.querySelector(userNameSelector);
-    this.userDescriptionElement = document.querySelector(
+    this._userDescriptionElement = document.querySelector(
       userDescriptionSelector
     );
+    this._profileAvatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
+    // console.log("getUserInfo: ", this.userDescriptionSelector);
     return {
       userName: this._userNameElement.textContent,
-      userDescription: this.userDescriptionElement.textContent
+      userDescription: this._userDescriptionElement.textContent,
     };
   }
 
-  setUserInfo({ title, description }) {
+  setUserInfo({ title, description, avatar }) {
     this._userNameElement.textContent = title;
-    this.userDescriptionElement.textContent = description;
+    this._userDescriptionElement.textContent = description;
+    this._profileAvatarElement.src = avatar;
```

Comment:
You should not change the avatar here, since the user info and avatar are controlled by the different forms. after receiving the initial info you can use both methods `setUserInfo` and `setAvatarInfo`